src/rpc/blockchain.cpp

```diff
@@ -1987,6 +1982,7 @@ static UniValue getblockstats(const JSONRPCRequest& request)
             maxfeerate = std::max(maxfeerate, feerate);
             minfeerate = std::min(minfeerate, feerate);
         }
+        tx_n++;
```

Comment:
looks fragile. (If some paths in the for loop `continue`, this would be skipped)